### DISCUSSION TITLE
How to chain multiple animations

### DISCUSSION THREAD

**Discussion by nexxai** [mdbootstrap/TW-Elements#1495] [Support from community]
I'm wondering if anyone can give me a hand chaining multiple animations.  

Specifically, I'm trying to take an element and have it fade in and up (using `[fade-in-up_2s_ease-in-out]`), wait for some period of time (let's say 5 seconds), and then have it fade up and out (using `[fade-out-up_2s_ease-in-out]`).  

I've tried using something like `fade-in-up_fade-out-up_2s_ease-in-out` but obviously that didn't work, and I don't know enough about CSS to really make heads or tails of what I'm trying to do.

**Comment by Trochonovitz** [comment]
Hi, I guess there is not any specific way to do this. For me works classic setTimeout - firstly removing old animate class then add new animate class.
